core/src/main/resources/hudson/slaves/JNLPLauncher/help.properties

```diff
@@ -21,14 +21,14 @@
 # THE SOFTWARE.
 
 blurb=\
-  Allows an agent to be connected to the Jenkins master whenever it is ready.<br>\
+  Allows an agent to be connected to the Jenkins controller whenever it is ready.<br>\
```

Comment:
> it's either a "Jenkins instance" or a "controller" (without Jenkins) usually

[citation needed]

A text quick search on jenkins-infra/jenkins.io shows that it's almost evenly split (i.e. half of "controller" occurrences have a "Jenkins" before them). Granted, some of that is influenced by security advisories where we put "Jenkins controller", but there's lots of occurrences elsewhere. For "Jenkins master"/"master" before it, there was also no clear preference -- since these changes neither add nor remove the "Jenkins" prefix.

Reply:
ok, thank you for the deep analyze, it's not a blocker anyway, more a personal preference :). I am fine with the your current version.